pom.xml

```diff
@@ -87,6 +87,12 @@
 	</repositories>
 
 	<dependencies>
+		<dependency>
```

Comment:
Could this and AWS dependencies be moved to profiles so that they are not loaded unless the profile is loaded.